### ISSUE TITLE
HTML element does not have a [lang] attribute

### ISSUE THREAD

**Issue by krzysiekpiasecki** [krzysiekpiasecki/Adminator-admin-dashboard#1]
<h1>Bug report</h1>

Accessibility matters.

# Current behavior

HTML element does not have a [lang] attribute.

# Expected behavior 

If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. 

# Environment
Platform: n/a
Browser:  n/a

For building issues:
Node:  n/a          
NPM:  n/a
